src/averell/readers/stichotheque.py

```diff
@@ -0,0 +1,84 @@
+import xml.etree.ElementTree as ETree
+
+from averell.utils import TEI_NAMESPACE as NS
+
+AOIDOS_NS = "{https://aoidos.ufsc.br/ns/1.0}"
+
+
+def parse_xml(xml_file):
+    """
+    XML TEI poem parser for 'Stichotheque' corpus. In this corpus, we have a list
+    of poems by author.
+    We read the data and find elements like title, author, etc with XPath
+    expressions.
+    Then, we iterate over the poem text and we look for each stanza and line
+    data.
+    :param xml_file: Path for the xml file
+    :return: Poem python dict with the data obtained
+    """
+    tree = ETree.parse(xml_file)
+    root = tree.getroot()
+    corpus_name = xml_file.parts[-4]
+    manually_checked = False
```

Comment:
Is it all automatically generated? Bummer.

Reply:
Yep, when the meter tag appears it is within the Aoidos Namespace.

Aoidos is their automatic scansion tool.